poe-tasks/get-modified-connectors.sh

```diff
@@ -29,14 +56,92 @@ all_changes=$(printf '%s\n%s\n%s\n%s' "$committed" "$staged" "$unstaged" "$untra
 filtered=$(printf '%s\n' "$all_changes" | grep -v -E "/${ignore_globs}")
 
 # 6) keep only connector paths
-connectors=$(printf '%s\n' "$filtered" | grep -E '^airbyte-integrations/connectors/[^/]+' )
+connectors_paths=$(printf '%s\n' "$filtered" | grep -E '^airbyte-integrations/connectors/(source-[^/]+|destination-[^/]+)(/|$)')
```

Comment:
This resolves a prior issue where non-connector paths inside the 'connectors' directory would get picked up as changes. Now, we only get directories that begin with either 'source-' or 'destination-'.

The PR now also includes dummy changes to these files as proof that they are _not_ picked up as connectors modified.

@davinchia discovered this issue when modifying the file: `airbyte-integrations/connectors/build.gradle`